app/controllers/export/base_controller.rb

```diff
@@ -0,0 +1,18 @@
+class Export::BaseController < PublicFacingController
```

Comment:
I wonder if we'd benefit more from subclassing from Admin::BaseController since that requires authentication and it looks like that does things we might not need: `https://github.com/alphagov/whitehall/blob/master/app/controllers/public_facing_controller.rb#L6-L8`

Reply:
This seems like a sensible thing to do 👍